app/src/main/java/kth/jjve/xfran/HomeScreenActivity.java

```diff
@@ -37,7 +37,8 @@ protected void onCreate(Bundle savedInstanceState) {
         WODDescription = findViewById(R.id.home_WODDescription);
         WODExercises = findViewById(R.id.home_WODExercises);
         Username = findViewById(R.id.home_username);
-        PlannedWorkouts = findViewById(R.id.home_rv);
+        //Todo: get the planned workouts of the day to the home view
+//        PlannedWorkouts = findViewById(R.id.home_rv);
```

Comment:
is the todo here normal? or is that the recycler view that we took away in the home screen)